_targets/build.project.ia32-generic

```diff
@@ -47,7 +47,7 @@ PAGE_MASK=$((~(SIZE_PAGE - 1)))
 BOOT_DEVICE="hd0"                 # Default boot device
 NAME_USER_SCRIPT="user.plo"       # User script name
 MAGIC_USER_SCRIPT=$((0xBAADF00D)) # User script magic value
-OFFS_USER_SCRIPT=$((0x12000))     # Disk image user script offset (in bytes)
+OFFS_USER_SCRIPT=$((0x12900))     # Disk image user script offset (in bytes)
```

Comment:
all of the disk addresses should be sector-aligned (512b here), IMHO better to move the kernel  to provide more space - that we wouldn't need to move it again in 2 weeks :P